src/runHandler.ts

```diff
@@ -91,12 +103,41 @@ export async function debugHandler(
   run.end()
```

Comment:
debugHandler need the same pre-process in runHandler. Because `runTest` assumes all the tests come from the same workspace folder

Reply:
yeah, that's a good idea